src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/XmlDecryptionTransform.cs

```diff
@@ -151,7 +151,7 @@ private void LoadStreamInput(Stream stream)
             XmlDocument document = new XmlDocument();
             document.PreserveWhitespace = true;
             XmlResolver resolver = (ResolverSet ? _xmlResolver : XmlResolverHelper.GetThrowingResolver());
-            XmlReader xmlReader = Utils.PreProcessStreamInput(stream, resolver, BaseURI!);
+            using XmlReader xmlReader = Utils.PreProcessStreamInput(stream, resolver, BaseURI!);
```

Comment:
It looks like stream could actually be coming in via a public entry point. Will this now dispose of a possibly user-provided Stream where it wasn't doing so before?

Reply:
Good catch. I had missed that. I can fix it up, but given the contribution bar, I guess I will just leave this PR closed.